client/packages/requisitions/src/RnRForms/DetailView/ContentArea.tsx

```diff
@@ -47,36 +48,55 @@ export const ContentArea = ({
   disabled,
 }: ContentAreaProps) => {
   const t = useTranslation('replenishment');
+  const ref = useRef<HTMLDivElement>(null);
 
   // TODO: move to backend, should join on item and sort by name!
   const lines = data.sort((a, b) => (a.item.name > b.item.name ? 1 : -1));
 
   return lines.length === 0 ? (
     <NothingHere body={t('error.no-items')} />
   ) : (
-    <Box flex={1} padding={2}>
+    <Box
+      flex={1}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        overflowX: 'unset',
+        overflowY: 'auto',
+        width: '100%',
+      }}
+      ref={ref}
+    >
       <GlobalStyles
         styles={{
+          thead: {
+            position: 'sticky',
+            top: 0,
+            backgroundColor: '#fff',
+            zIndex: 999,
+          },
           '.sticky-column': {
             backgroundColor: '#fff',
             position: 'sticky',
             zIndex: 99,
           },
           '.first-column': {
+            left: 0,
             position: '-webkit-sticky',
-            left: 16,
             width: 80,
           },
           '.second-column': {
-            position: '-webkit-sticky',
-            left: 88,
-            minWidth: '300px',
             borderRight: '1px solid blue',
+            left: 72,
+            minWidth: '300px',
+            position: '-webkit-sticky',
```

Comment:
I think we might need the column widths...

https://github.com/user-attachments/assets/0e042b4c-c2a9-48de-b981-38f0173b6799